uber/site_sections/summary.py

```diff
@@ -215,23 +215,7 @@ def exceeds_threshold(start_time, attendee):
         return {'flagged': flagged}
 
     def setup_teardown_neglect(self, session):
```

Comment:
This page previously had two sections.  One of those sections is still relevant to the core plugin, since it refers to setup/teardown shifts.  The other section has been moved to its own page in the ``hotel`` plugin.